### PR TITLE
Adds aliasing support for TortoiseGit commands

### PR DESCRIPTION
**Pull request by eamodio**
Also allows configuration of TortoiseGit commands

### CODE REVIEW

File: src/GitTabExpansion.ps1 [hunk 1]
```diff
@@ -194,7 +194,7 @@ function GitTabExpansionInternal($lastBlock) {
     # Handles tgit <command> (tortoisegit)
     if ($lastBlock -match "^$(Get-AliasPattern tgit) (?<cmd>\S*)$") {
         # Need return statement to prevent fall-through.
-        return $tortoiseGitCommands | Where-Object { $_ -like "$($matches['cmd'])*" }
+        return $Global:TortoiseGitSettings.TortoiseGitCommands.Keys.GetEnumerator() | sort | Where-Object { $_ -like "$($matches['cmd'])*" }
```
Comment (dahlbyk) [inline src/GitTabExpansion.ps1]: s/sort/Sort-Object/

(also in TortoiseGit.ps1)

File: src/TortoiseGit.ps1 [hunk 2]
```diff
@@ -11,60 +11,74 @@ function private:Get-TortoiseGitPath {
...
+    "config" = "settings";
+    "stash" = "stash";
+    "stashapply" = "stashapply";
+    "stashsave" = "stashsave";
+    "subadd" = "subadd";
+    "subsync" = "subsync";
+    "subupdate" = "subupdate";
+    "switch" = "switch";
+    "checkout" = "switch";
+    "sync" = "sync";
```
Comment (dahlbyk) [inline src/TortoiseGit.ps1]: Would it make sense to alias `fetch` to `sync`?

File: src/TortoiseGit.ps1 [hunk 3]
```diff
@@ -11,60 +11,74 @@ function private:Get-TortoiseGitPath {
...
-"stashapply",
-"subadd",
-"subupdate",
-"subsync",
-"reflog",
-"refbrowse",
-"sync",
-"repostatus"
-) | sort
+}
```
Comment (dahlbyk) [inline src/TortoiseGit.ps1]: EOL @ EOF, please!

(Tip: we have a [.editorconfig](http://editorconfig.org/) that will enforce this for supported editors.)

File: src/GitTabExpansion.ps1 (hunk 1 above)
Comment (rkeithhill) [inline src/GitTabExpansion.ps1]: Good catch.  PowerShell Core running on Linux or macOS does not have the sort alias defined for Sort-Object.  So calling sort would execute the native sort utility.

File: src/TortoiseGit.ps1 (hunk 3 above)
Comment (eamodio) [inline src/TortoiseGit.ps1]: Fixed :)

File: src/TortoiseGit.ps1 (hunk 2 above)
Comment (eamodio) [inline src/TortoiseGit.ps1]: Added.

File: src/GitTabExpansion.ps1 (hunk 1 above)
Comment (eamodio) [inline src/GitTabExpansion.ps1]: Fixed.

**Comment by dahlbyk** [comment]
:100: :clap: :tada:

**Pull request merged**